app/controllers/merchant/orders_controller.rb

```diff
@@ -5,4 +5,13 @@ def show
     @merchant = Merchant.find(current_user[:merchant_id])
   end
 
+  def update
```

Comment:
This is basically all I needed to do to wrap up the merchant fulfillment user stories